README.md

```diff
@@ -156,6 +156,83 @@ To have the linter also _fix_ the violations run: `npm run lint:fix`.
+- [`bundle/android/strings.xml`](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/bundle/android/strings.xml)
+- [`bundle/ios/GutenbergNativeTranslations.swift`](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/bundle/ios/GutenbergNativeTranslations.swift)
+
+These files are generated via the `i18n:update` NPM command, and like translations, they are also produced when generating the bundle.
+
+### NPM commands
+- `npm run i18n:update`: Downloads optimized translations and generate localization strings files for all plugins. **NOTE:** This command is attached to `bundle` NPM command via `prebundle:js`, so it will be automatically executed when generating a bundle.
+- `npm run i18n:check-cache`: Downloads un-optimized translations for plugins that don't have a cache folder. **NOTE:** This command is attached to dependency installation via `postinstall`, so it will be automatically executed when installing dependencies.
+
+### How to add a new plugin
+1. Identify the i18n domain, which usually matches the plugin's name (i.e. `jetpack`).
+2. Identify the path to the plugin source code (i.e. `./jetpack/projects/plugins/jetpack/extensions`).
+3. Append the plugin's name to the arguments of `i18n:check-cache` NPM command.
+4. Append the plugin's name and source code path to the arguments of `i18n:update` NPM command.
+5. Add the i18n domain of the plugin and the callback for getting translation to the [editor initialization](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/src/index.js).
+*Example:*
+```
+import { getTranslation as getJetpackTranslation } from './i18n-translations/jetpack';
+...
+
+const pluginTranslations = [
+	{
+		domain: 'jetpack',
+		getTranslation: getJetpackTranslation,
+	},
+	...
+];
+```
+
+## Caveats
+- Strings that are only used in the native version, and reference a [context](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#disambiguation-by-context), won't be included in the localization strings files hence, they won't be translated. This is a limitation in the format of the localization strings files.
+- Localization strings files don’t support domains, so the strings extracted from plugins that are only used in the native version, will be unified in the same file, which might involve string conflicts.
+
+## Troubleshooting
+
+### A string is missing the translation
+This can be produced by several causes, check the following steps in order to identify the source:
+- Verify that the string uses the `__` i18n function or similar ([reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/i18n/README.md)).
+- Verify warnings in the output when running `i18n:update` NPM command, especially the following ones:
```

Comment:
Something like "Check here" reads better to me than "Verify". This is definitely not a big deal at all, so feel free to leave it as-is if you prefer.

```suggestion
- Check for warnings in the output when running `i18n:update` NPM command, especially the following ones:
```